### ISSUE TITLE
Installation error for sits package on MPC

### ISSUE THREAD

**Issue by ritviksahajpal** [e-sensing/sits#905]
**Describe the bug**
First, thanks for an impressive package! I am reporting a bug related to an installation error for sits package on MPC

**To Reproduce**
install.packages("sits")

**If reporting a change from previous versions**

Please read https://cran.r-project.org/web/packages/sits/news/news.html first.

**Additional context**
Installation of sits package requires the torch package which exits with a non-zero error and subsequently sits package installation fails too.

```
Warning message in install.packages("torch"):
“installation of package ‘torch’ had non-zero exit status”
Updating HTML index of packages in '.Library'

Making 'packages.html' ...
 done
```

<details>
R version 4.1.3 (2022-03-10)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: Ubuntu 22.04 LTS

Matrix products: default
BLAS/LAPACK: /srv/conda/envs/notebook/lib/libopenblasp-r0.3.21.so

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       
 [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   
 [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          
[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] fansi_1.0.3     crayon_1.5.1    digest_0.6.29   utf8_1.2.2     
 [5] IRdisplay_1.1   repr_1.1.4      lifecycle_1.0.1 jsonlite_1.8.0 
 [9] evaluate_0.16   pillar_1.8.1    rlang_1.0.5     cli_3.4.0      
[13] uuid_1.1-0      vctrs_0.4.1     IRkernel_1.3    tools_4.1.3    
[17] glue_1.6.2      fastmap_1.1.0   compiler_4.1.3  base64enc_0.1-3
[21] pbdZMQ_0.3-7    htmltools_0.5.3
</details>

**Comment by ritviksahajpal** [comment]
I realize that torch installation is not a sits issue per se, but I am asking in case you have seen this issue before

**Comment by ritviksahajpal** [comment]
cross-listed on MSPC github issues page: https://github.com/microsoft/PlanetaryComputer/issues/176

**Comment by gilbertocamara** [comment]
Dear @ritviksahajpal 

We use the R version of torch in MPC for our work without problems. Please follow the installation procedures described in https://e-sensing.github.io/sitsbook/setup.html. 

We strongly suggest you use an Ubuntu virtual machine. There are a number of limitation of Windows platforms for the R environment, most notably linked to parallelisation. 

Please use an Ubuntu VM, install R and RStudio, and then install “sits” from CRAN in the usual way:

`> install.packages(“sits”, dependencies = TRUE)`

This should install everything, including “torch”. After installation is finished, please run

`> torch::install_torch()`

In case of problems, please contact us.

**Comment by ritviksahajpal** [comment]
Thank you so much! I am probably doing something wrong as I tried what you suggested and could not get it to work. First, I started a terminal from the Launcher window on MPC. Next, I downloaded the shell script using wget and then tried to run it using sh. However, I do not have sudo access on MPC and the shell script does not run without sudo access. As a MPC novice, I am probably not setting up the virtual machine correctly, but any guidance would be much appreciated. Here is a screenshot of my MPC session:

![image](https://user-images.githubusercontent.com/2277375/215542462-51a33349-7716-4fa1-8554-19ef0c4dbd93.png)
